### PR TITLE
Remove jquery dependency

### PR DESCRIPTION
**Pull request by jrschumacher**
Remove jQuery dependency. Depends on PR #35

### CODE REVIEW

**Comment by jrschumacher** [comment]
@RobQuistNL did you have a chance to check this out?

File: manifest.json [hunk 1]
```diff
@@ -22,13 +22,14 @@
     },
 
     "web_accessible_resources": [
-        "images/jira.png"
+        "images/jira.png",
+        "src/templates.js"
```
Comment (RobQuistNL) [inline manifest.json:26]: Is this a remnant of something extra you were working on?

File: manifest.json [hunk 2]
```diff
@@ -22,13 +22,14 @@
...
     ],
 
     "content_scripts": [
         {
             "matches": ["*://github.com/*"],
-            "js": ["vendor/jquery-3.3.1.min.js", "src/content.js"],
+            "js": ["src/content.js"],
```
Comment (RobQuistNL) [inline manifest.json:32]: :clap: :clap: :clap:

File: src/content.js [hunk 3]
```diff
@@ -1,244 +1,337 @@
...
+}
 
 function checkPage() {
     let url = window.location.href;
-    if (url.match(/github\.com\/(.*)\/(.*)\/pull\//) != null) {
-        setTimeout(function() {
-            handleCommitsTitle();
-            handlePrPage();
-        }, 200); //Small timeout for dom to finish setup
+    if (url.match(GITHUB_PAGE_PULL) != null) {
```
Comment (RobQuistNL) [inline src/content.js:208]: :1st_place_medal:

File: src/content.js [hunk 4]
```diff
@@ -1,244 +1,337 @@
...
-            );
+    try {
+        const result = await sendMessage({ query: 'getTicketInfo', jiraUrl, ticketNumber })
+        if (result.errors) {
+            throw new Error(result.errorMessages);
         }
-    );
+        loadingElement.innerHTML = headerBlock(ticketNumber, result.fields);
+    } catch(e) {
+        console.error('Issue fetching data', e)
```
Comment (RobQuistNL) [inline src/content.js:273]: Might want to rename "Issue" to "Error" or "Problem", as "issue" is kind of ambiguous when we're talking about "issues in Jira" :stuck_out_tongue_winking_eye:

File: manifest.json (hunk 1 above)
Comment (jrschumacher) [reply manifest.json:26]: Gah. It is :)

File: src/content.js (hunk 4 above)
Comment (jrschumacher) [reply src/content.js:273]: Ha! No joke.

**Comment by jrschumacher** [comment]
@RobQuistNL great review and glad you caught those. I've resolved them. I also added a changelog.

**Comment by RobQuistNL** [comment]
Sorry for the late merge @jrschumacher ! Busy days :( 

Thanks a ton for your contribution <3

I'll see if I can merge some other stuff in and then I'll release a new version on the store.

Cheers!

**Pull request merged**